app/decorators/change_report_decorator.rb

```diff
@@ -70,7 +70,7 @@ def last_paycheck
   end
 
   def client_name
-    [member&.first_name, member&.last_name].compact.join(" ") if member
+    member&.full_name
```

Comment:
I never knew about the Safe Navigation Operator. Def going to start using it.

Reply:
It's relatively new (ruby 2.3.0)! and super useful.